extensions/wikia/AdEngine/js/AdEngine2.js

```diff
@@ -68,11 +80,17 @@ define('ext.wikia.adEngine.adEngine', [
 					provider = providerList.shift();
 
 					if (!provider) {
-						return fillInSlotUsingProvider(slotname, adProviderNull, noop);
+						delete slot.success;
+
+						if (slot.error) {
```

Comment:
I'm not confident about this - If some developer will misuse slot.error callback then he will get error right away. The change you propose is solid in terms of there won't be any errors in production, but upon misuse it would not trigger any errors and it would be hard to find a spot why function is not triggered, right?